devsetup/Makefile

```diff
@@ -60,6 +60,9 @@ EDPM_COMPUTE_ADDITIONAL_HOST_ROUTES ?= '[]'
 TLS_ENABLED ?= false
 DNS_DOMAIN ?= localdomain
 
+BARBICAN_SERVICE_ENABLED ?= false
+MANILA_SERVICE_ENABLED ?= false
+
```

Comment:
right now; both these services are enabled on the standalone deployment; your change will break that: https://github.com/openstack-k8s-operators/install_yamls/blob/ff5075d356345f9e7e945ac53a164aebae055c52/devsetup/standalone/openstack.sh#L23-L24

Please default to True and disable these locally if you wish.

Reply:
Thanks, I made the change